CONTRIBUTING.md

```diff
@@ -14,38 +14,54 @@ Also confirm that you can access the Phantom web app at `https://{phantom_VM_IP_
 
 NOTE: If you are consistently getting timeouts connecting to the Phantom VM, try disconnecting from the VPN and re-attempting.
 
+### Style linter
+
+When you open a PR a series of style checks will run. See the [pre-commit-config.yaml](.pre-commit-config.yaml) file to see a list of the projects involved in this automation. If your code does not pass the style checks, the PR will not be allowed to merge. Many of the style rules can be corrected automatically by running a simple command once you are satisfied with your change:
+
+```bash
+make style
+```
+
+This will output a diff of the files that were changed. Once these have been corrected and re-pushed, the PR checks should pass.
+
+You can optionally also choose to have these checks / automatic adjustments
```

Comment:
Probably don't need the both words `optionally` and `also` at the beginning of this sentence